src/site/stages/build/process-cms-exports/transformers/helpers.js

```diff
@@ -50,12 +50,33 @@ module.exports = {
   /**
    * Takes a string and applies the following:
    * - Transforms escaped unicode to characters
-   * - Unescapes HTML entities
    *
    * @param {string}
    * @return {string}
    */
   getWysiwygString(value) {
     return unescape(value);
   },
+
+  /**
+   * Finds the property name containing the UUID.
+   *
+   * @param {Object} parent - The entity to look in
+   * @param {string} uuid - The UUID to search for
+   * @return {string} - The snake_cased property name the UUID is
+   *                    found in
+   * @return {undefined} - If the UUID is not found
+   */
+  getRawParentFieldName(parent, uuid) {
+    let rawParentFieldName;
+    Object.keys(parent).forEach(key => {
+      if (!Array.isArray(parent[key])) return;
+
+      parent[key].forEach(prop => {
+        if (prop.target_uuid === uuid) rawParentFieldName = snakeCase(key);
```

Comment:
Would it make sense to break out of this loop once we've set `rawParentFieldName`, or do we want to allow for follow-up values to override it?

Reply:
Yeah, I mean, if the child entity is found in two or more properties, `rawParentFIeldName` is going to be wrong in one, so whichever way we go, it'll...not be great. :grimacing:

Reply:
Well, I guess let's hope that the entity tree remains a tree :crossed_fingers: